common/logisticspipes/proxy/specialinventoryhandler/StorageDrawersInventoryHandler.java

```diff
@@ -102,8 +117,11 @@ public ItemStack getMultipleItems(ItemIdentifier itemIdent, int count) {
 	public HashMap<ItemIdentifier, Integer> getItemsAndCount() {
 		HashMap<ItemIdentifier, Integer> result = new HashMap<ItemIdentifier, Integer>();
 		for(int i=0; i < _drawer.getDrawerCount(); i++) {
+			if (!_drawer.isDrawerEnabled(i))
+				continue;
+
 			IDrawer drawer = _drawer.getDrawer(i);
-			if(drawer != null && drawer.getStoredItemCopy() != null) {
+			if(drawer != null && !drawer.isEmpty()) {
 				int count = drawer.getStoredItemCount();
 				ItemIdentifier ident = ItemIdentifier.get(drawer.getStoredItemCopy());
```

Comment:
isEmpty() is defined by the API such that the drawer must actually be unbound.  It's equivalent in this case.  I have a question on this point though -- what is the correct behavior when the drawer is bound to an item but has no count (because it's locked)?  Right now it will still return that that item with the list.